mongodbatlas/mongodbatlas.go

```diff
@@ -61,7 +61,7 @@ type Client struct {
 	X509AuthDBUsers                  X509AuthDBUsersService
 	ContinuousSnapshots              ContinuousSnapshotsService
 	ContinuousRestoreJobs            ContinuousRestoreJobsService
-	AtlasCheckpoints                 AtlasCheckpointService
+	AtlasCheckpoints                 CheckpointService
```

Comment:
missed the rename here, can we also make CheckpointService -> CheckpointsService?